packaging/msix/generate-icons.py

```
@@ -0,0 +1,154 @@
+  Assets/amux.ico                — Windows .exe multi-size icon
+"""
+
+from __future__ import annotations
+from pathlib import Path
+from typing import Optional
+
+try:
+    from PIL import Image, ImageDraw, ImageFont
+except ImportError:
+    print("Pillow not installed. Install with: pip install Pillow")
+    raise SystemExit(1)
```

Ruff (0.15.10)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)